### ISSUE TITLE
Query password change does not when passed via command line parameter

### ISSUE THREAD

**Issue by WolverinDEV** [TeaSpeak/TeaSpeak#653]
```
[2020-03-10 09:12:50] [INFO] GEN | Updating server admin query password to "XXXXXXX"
[2020-03-10 09:12:50] [ERROR] GEN | Failed to update server admin query password! Login does not exists!
```

**Comment by dwwe2017** [comment]
It would be desirable that this is also possible with teastart.sh, e.g. B. ./teastart.sh -p password

**Comment by WolverinDEV** [comment]
Its already possible:
`teastart.sh start -p <password>`

**Comment by dwwe2017** [comment]
Is that already fixed in version 1.4.10-beta-3 or only in beta-4?
